crates/ide-completion/src/tests/special.rs

```diff
@@ -893,3 +902,89 @@ fn f() {
         "#]],
     );
 }
+
```

Comment:
not sure if putting tests here (special.rs) is better than https://github.com/rust-lang/rust-analyzer/blob/45ec315e01dc8dd1146dfeb65f0ef6e5c2efed78/crates/ide-completion/src/tests.rs